test/strategies/base.ts

```diff
@@ -69,6 +69,74 @@ describe('Strategy', () => {
       expect(pullRequest?.version?.toString()).to.eql('2.3.4');
       snapshot(dateSafe(pullRequest!.body.toString()));
     });
+    it('updates extra files', async () => {
+      const strategy = new TestStrategy({
+        targetBranch: 'main',
+        github,
+        component: 'google-cloud-automl',
+        extraFiles: [
+          '0',
+          'foo/1.~csv',
+          'foo/2.bak',
+          'foo/baz/bar/',
+          '/3.java',
+          '~/4.md',
+          '~/./5',
```

Comment:
I don't think there's much of a point of allowing `~` if we're just stripping it away. Let's remove the `~` handling and tests.

Reply:
Agreed, dropped support in f9968f8025443b3aa062794e6a37b21d4e2e56b9